NameLayerMain/src/vg/civcraft/mc/namelayer/misc/Mercury.java

```diff
@@ -1,14 +1,17 @@
 package vg.civcraft.mc.namelayer.misc;
 
 import vg.civcraft.mc.mercury.MercuryAPI;
+import vg.civcraft.mc.namelayer.NameLayerPlugin;
 
 public class Mercury {
 
 	/**
-	 * Invalidates a group across all servers.
+	 * Sends a message to all servers.
 	 * @param message- The message to send.
 	 */
-	public static void invalidateGroup(String message){
-		MercuryAPI.sendGlobalMessage(message, "namelayer");
+	public static void message(String message){
+		if (NameLayerPlugin.isMercuryEnabled()){
+			MercuryAPI.sendGlobalMessage(message, "namelayer");
+		}
```

Comment:
I tend to agree with Max here. For consistency and easy maintenance it would be best if each type of message had its own method stub here. Yes it's a bit of painful extra coding, but should pay dividends in maintaining and discovery missed or malfunctioning mercury message exchanges.